### DISCUSSION TITLE
Local vs Symbol

### DISCUSSION THREAD

**Discussion by ZakolkaTV** [Somfic/EliteVA#15] [Q&A] [answered]
Thank you very much for your work, it is priceless.
Help me understand should these variables be shown?
EliteAPI.SelectedWeapon.Local
EliteAPI.SelectedWeapon.Symbol

**Comment by Somfic** [comment, marked as answer]
Hi @ZakolkaTV,

I am not entirely sure what you are asking, but I can explain the difference between `Local` and `Symbol`, and why they're showing up. This has to do with languages and translations.

Take for example part of this journal event:

```json
{ 
	"Government": "$government_PrisonColony;", 
	"Government_Localised": "Colonie pénitentiaire" 
}
```

When the game has to output text that is different between different languages, for example for commodities, factions and other things, the game outputs both a `symbol` and a `localised` version of the text.

The `symbol` is the culture-invariant version of the text, in this case: `$government_PrisonColony;`. Though a bit harder to read, it is the non-translated version of the value.

The `local` is the translated version of the value, in this case: `Colonie pénitentiaire`. As you can imagine, if I set my game language to Russian, this value would change. Important to note is that the value in `symbol` would *not* change.

I hope this explains why both `Local` and `Symbol` versions of `SelectedWeapon` exist. 

If you are planning on sharing your VoiceAttack profile with people that might not have the same lanuage as you, I would use the `symbol` value in your if statements, since that value does not change between language settings. If however this is a private VoiceAttack profile, you can use both `symbol` and `local`, since you probably will not change your own language.

I hope this helps! Let me know if you have more questions (or if I undestood your question wrong)

:)

**Comment by ZakolkaTV** [comment]
Thanks for the detailed answer and I apologize for my English =)

Thanks for the explanation. Now I understand what “Local” and “Symbol” are. But I still have a question. What is a "EliteAPI.SelectedWeapon" and how can I use it?

**Reply by Somfic** [reply]
It's for the Odyssey DLC. When you are on-foot on planets you have a weapon, like a rifle or pistol. This variable denotes which one, if any, you have selected.

**Reply by ZakolkaTV** [reply]
Oh, how I didn’t realize it myself, it was simple. Thanks for the answer.
